viewing_party/party.py

```diff
@@ -1,23 +1,201 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    new_movie = {}
+    
+    if not title or not genre or not rating:
+        return None
+    
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    return new_movie

```

Comment:
One way to shorten this could be like the following: 
```python
if title and genre and movie:
     return {
         "title": title,
         "genre": genre,
         "rating": rating
     }
     
return None
```